testing/src/cypress/page-objects/Institution-objects/DashboardInstitutionObject.ts

```diff
@@ -13,7 +13,8 @@ export enum CredentialTypes {
 
 export enum ProgramLengthOptions {
   WeeksToLessThanYear = "12WeeksToLessThan1Year",
-  OneToTwoYears = "1YearToLessThan2Years",
+  WeeksTo59Weeks = "52WeeksTo59Weeks",
+  WeeksToTwoYears = "60WeeksToLessThan2Years",
```

Comment:
https://github.com/bcgov/SIMS/pull/2810/files#r1483399939

Reply:
We don't need to keep the original values since the change (for the dropdown) is required prior to Part-Time release.